windows_containers/windows-containers-release-notes-3-x.adoc

```diff
@@ -21,13 +21,34 @@ ifndef::openshift-origin[]
 
 {productwinc} is provided and available as an optional, installable component. Windows Container Support for Red Hat OpenShift is not part of the {product-title} subscription. It requires an additional Red Hat subscription and is supported according to the link:https://access.redhat.com/support/offerings/production/soc/[Scope of coverage] and link:https://access.redhat.com/support/offerings/production/sla[Service level agreements].
 
-You must have this separate subscription to receive support for Windows Container Support for Red Hat OpenShift. Without this additional Red Hat subscription, deploying Windows container workloads in production clusters is not supported. You can request support through the link:http://access.redhat.com/[Red Hat Customer Portal]. 
+You must have this separate subscription to receive support for Windows Container Support for Red Hat OpenShift. Without this additional Red Hat subscription, deploying Windows container workloads in production clusters is not supported. You can request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
 
 For more information, see the Red Hat OpenShift Container Platform Life Cycle Policy document for link:https://access.redhat.com/support/policy/updates/openshift#windows[Red Hat OpenShift support for Windows Containers].
 
-If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support. 
+If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support.
 endif::openshift-origin[]
 
+[id="wmco-3-1-2"]
+== Release notes for Red Hat Windows Machine Config Operator 3.1.2
+
+Issued: 2022-11-09
+
+The WMCO 3.1.2 is now available with bug fixes. The components of the WMCO were released in link:https://access.redhat.com/errata/RHBA-2022:7076[RHBA-2022:7076].
+
+[id="wmco-3-1-2-new-features"]
+=== New features
+
+[id="wmco-3-1-2-win2022"]
+==== Windows Server 2022 support
+
+With this release, Windows Server 2022 now supports VMware vSphere.
+
+[id="wmco-3-1-2-bug-fixes"]
+=== Bug fixes
+
+* Previously, Windows Containers on Windows Nodes were assigned the wrong DNS Server IP. Therefore, the DNS resolution did not work. This bug fix removes the hard-coded cluster DNS information and parametrizes the value as a command-line argument.
```

Comment:
`Previously, Windows Containers on Windows Nodes were assigned the wrong DNS Server IP if a non-default cluster DNS was specified during the cluster install.`